css-layout-classes/styles.css

```diff
@@ -0,0 +1,32 @@
+.container {
+  margin: auto;
+  width: 75%;
+}
+
+.row {
+  display: flex;
+}
+
+.col-3 {
+  flex-basis: calc (100% / 3);
+}
+
+.col-4 {
+  flex-basis: 25%;
+}
+
+.justify-center {
+  justify-content: center;
+}
+
+.row-3 {
```

Comment:
What is this class name meant to mean? This can probably be renamed to better describe its purpose